example/extending/newlang/5-hello-python/build.mill

```diff
@@ -0,0 +1,138 @@
+// == Basic Python Pipeline 
+
+// This Example demonstrates integration of https://www.python.org[Python]
+// compilation into a Mill build to compile https://www.python.org[Python] Scripts. Mill
+// does not come bundled with https://www.python.org[Python] integration, so here we begin setting
+// one up from first principles using the Basic `Python3`(Already Present in OS) command Line
+
+
+// === Installing Python Pip
+//
+// First, we need to use the `Python3` CLI tool to install `Pip` 
+// using the official https://bootstrap.pypa.io/get-pip.py[get-pip.py] Code:
+
+package build
+import mill._
+
+def installPip: T[PathRef] = Task {
+  val pipFile = Task.dest / "get-pip.py"
+  val pipPath = os.home / ".local" / "bin" / "pip3"
+
+  if (!os.exists(pipPath)) {
+    os.call(Seq("wget", "-q", "https://bootstrap.pypa.io/get-pip.py", "-O", s"$pipFile"))
```

Comment:
We should not need to install pip because Python3 bundles pip for all versions >3.4 and above. Let's just use the bundled one